README.md

```diff
@@ -39,34 +39,117 @@ jobs:
       - name: Run Django Tests
         run: docker-compose -f local.yml exec -T django coverage run --rcfile=.pre-commit/setup.cfg -m pytest --disable-pytest-warnings;
       - name: Print Coverage
-        run: docker-compose -f local.yml exec -T django coverage report
-      - name: Print Coverage to txt file
         run: docker-compose -f local.yml exec -T django coverage report > coverage.txt
       - name: Tear down the Stack
         run: docker-compose -f local.yml down
       - name: Update Coverage badge
         uses: Orfium/coverage-badge-creator@master
         with:
-          coverage_file: coverage.txt
+          coverage_file: "coverage.txt"
           badge_name: "code-cov-prod.svg"
           upload_coverage_file: true
           bucket_name: "orfium-badges-bucket"
           aws_access_key: ${{ secrets.BADGES_AWS_ACCESS_KEY_ID }}
           aws_secret_key: ${{ secrets.BADGES_AWS_SECRET_ACCESS_KEY }}
           coverage_percentage_regex: ${{ secrets.COVERAGE_PERCENTAGE_REGEX }}
         if: github.ref == 'refs/heads/develop'
-      
+
 ```
 
 ## Input Variables
 
-| Name                        | Description                                  | Default |
-| --------------------------- | ---------------------------------------------| ------- |
-| `coverage_file`             | Path to coverage file                        | -       |
-| `badge_name`                | Name of the badge file                       | -       |
-| `upload_coverage_file`      | Upload coverage file too                     | False   |
-| `bucket_name`               | Name of the bucket to upload the badge to    | -       |
-| `aws_access_key`            | AWS Access Key                               | -       |
-| `aws_secret_key`            | AWS Secret Key                               | -       |
-| `coverage_percentage_regex` | Regex to use in order to retrieve coverage   | -       |
+| Name                            | Description                                     | Default   | Required  |
+| ------------------------------- | ----------------------------------------------- | --------- | --------- |
+| `coverage_file`                 | Path to coverage file (txt or json reports)     | -         | Yes       |
+| `badge_name`                    | Name of the badge file                          | -         | Yes       |
+| `upload_coverage_file`          | Upload coverage file too                        | False     | No        |
+| `bucket_name`                   | Name of the bucket to upload the badge to       | -         | Yes       |
+| `aws_access_key`                | AWS Access Key                                  | -         | Yes       |
+| `aws_secret_key`                | AWS Secret Key                                  | -         | Yes       |
+| `coverage_percentage_regex`     | Regex or path to use in order to get coverage   | -         | Yes       |
```

Comment:
I suggest that we keep the `coverage_percentage_regex` variable and make it optional. We could create another variable named `coverage_json` which would also be optional.